app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -1040,6 +1036,119 @@ class PlayerFragment : BaseFragment(), OnlinePlayerOptions {
         }
     }
 
+    private fun setupDescription(
+        descTextView: TextView,
+        description: String,
+        exoPlayer: ExoPlayer
+    ) {
+        // detect whether the description is html formatted
+        if (description.contains("<") && description.contains(">")) {
+            descTextView.movementMethod = LinkMovementMethod.getInstance()
+            descTextView.text = HtmlParser.createSpannedText(
+                description,
+                LinkHandler { link ->
+                    // check if the link is a youtube link
+                    if (link.contains("youtube.com") || link.contains("youtu.be")) {
+                        // check if the link is a video link
+                        val videoId = getVideoIdIfVideoLink(link)
+                        if (!videoId.isNullOrEmpty()) {
+                            // check if the video is the current video
+                            if (videoId == this.videoId) {
+                                // get the time from the link
+                                val time = link.substringAfter("t=").substringBefore("&")
+                                // check if the time is valid
+                                if (time.isNotEmpty() && time != link) {
+                                    // get the time in seconds
+                                    val timeInSeconds = getTimeInMillis(time)
+                                    if (timeInSeconds != -1L) {
+                                        // seek to the time
+                                        exoPlayer.seekTo(timeInSeconds)
+                                    }
+                                }
+                                // youtube link without time
+                                // open new player
+                                playNextVideo(videoId)
```

Comment:
That apparently is still being called even though the player previously seeked to another position in line 1065, so the video will be reloaded although we already figured out it's only a timestamp which is not what we want to achieve.

Reply:
Sorry, here I forgot an else condition. I must fix this by merging both  `playNextVideo(videoId)`.

Reply:
Fortunately video was not reloading because in `playNextVideo` it checks for difference.